books/free-programming-books-sr.md

```diff
@@ -1,8 +1,45 @@
 ### Index
 
 * [C](#c)
+* [C++](#cpp)
+* [CSS](#css)
+* [HTML](#html)
+* [Java](#java)
+* [JavaScript](#javascript)
+* [Python](#python)
 
 
 ### C
 
 * [Beej-ov vodič za mrežno programiranje - Korištenje Internet soket-a](https://web.archive.org/web/20181008134854/http://users.teol.net:80/~mvlado/sockets/) - Brian "Beej Jorgensen" Hall, Maksimović Darko (HTML) *(:card_file_box: archived)*
+
+
+### C++
+
+* [Programski jezik C++ sa rešenim zadacima (2010)](https://singipedia.singidunum.ac.rs/izdanje/40777-programski-jezik-c-sa-resenim-zadacima) -  Ranko Popović, Zona Kostić, Univerzitet Singidunum (PDF)
+
+
+### CSS
+
+* [Web dizajn: HTML, CSS i JavaScript (2020)](https://singipedia.singidunum.ac.rs/izdanje/42767-web-dizajn-html-css-i-javascript) - Nenad Kojić, Univerzitet Singidunum (PDF)
```

Comment:
```suggestion
* [Web dizajn: HTML, CSS i JavaScript](https://singipedia.singidunum.ac.rs/izdanje/42767-web-dizajn-html-css-i-javascript) - Nenad Kojić (PDF)
```
we only include the year when the book is old enough that it might be out of date.